rst2pdf/tests/conftest.py

```diff
@@ -1,9 +1,8 @@
-"""
-Copyright (c) 2020, Stephen Finucane <stephen@that.guru>
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020, Stephen Finucane <stephen@that.guru>
 
+"""
```

Comment:
The same question as above, what does it mean for an individual to have copyright on a file that's also MIT licensed by the project?

Reply:
@stephenfin do you know the answer to this?